src/common/imageio_rawspeed.h

```diff
@@ -25,6 +25,8 @@ extern "C" {
 #include "common/image.h"
 #include "common/mipmap_cache.h"
 
+#define ADOBE_COEFF_FACTOR 10000.0f
```

Comment:
```suggestion
#define ADOBE_COEFF_DENOM 10000
```

Reply:
Btw, shouldn't this macro go better into a more general header like image.h as it's used in image_dng.h rather than pulling image_rawspeed.h there?

Reply:
Done, and moved into `imageio.h`, better indeed.